### PR TITLE
Fix translation with lupdate

### PR DESCRIPTION
**Pull request by signedav**
By installing qt5-default in advance.

### CODE REVIEW

**Comment by qfield-fairy** [comment]
Uploaded test apks for
 - [**armv7**](https://download.opengis.ch/qfield/ci-builds/qfield-dev-1293-3baba8dd496740e41d7ba01d27b28f224c4e51f3-armv7.apk)
 - [**x86**](https://download.opengis.ch/qfield/ci-builds/qfield-dev-1293-3baba8dd496740e41d7ba01d27b28f224c4e51f3-x86.apk)
 - [**arm64_v8a**](https://download.opengis.ch/qfield/ci-builds/qfield-dev-1293-3baba8dd496740e41d7ba01d27b28f224c4e51f3-arm64_v8a.apk)
 - [**x86_64**](https://download.opengis.ch/qfield/ci-builds/qfield-dev-1293-3baba8dd496740e41d7ba01d27b28f224c4e51f3-x86_64.apk)

**Comment by m-kuhn** [comment]
Can this be put to the other aptions? https://github.com/opengisch/QField/blob/129aa1602123faa191f76a01076e5bae40406468/scripts/docker-build.sh#L26

File: scripts/docker-build.sh [hunk 1]
```diff
@@ -23,7 +23,7 @@ if [[ -z $APP_VERSION ]] && [[ -z $APP_VERSION_CODE ]]; then
   exit 2
 fi
 
-apt update && apt install -y zip bc cmake ninja-build jq
+apt update && apt install -y zip bc cmake ninja-build jq ; apt install qt5-default
```
Comment (suricactus) [inline scripts/docker-build.sh:26]:
```suggestion
apt update && apt install -y zip bc cmake ninja-build jq qt5-default
```
Why not like this?

Comment (signedav) [reply scripts/docker-build.sh]: Because I didn't looked well. But it fails anyway. What about putting it into translation.yml?

**Comment by qfield-fairy** [comment]
Uploaded test apks for
 - [**armv7**](https://download.opengis.ch/qfield/ci-builds/qfield-dev-1293-e1246c8824a83c760b962d970625d356b711ae8e-armv7.apk)
 - [**x86**](https://download.opengis.ch/qfield/ci-builds/qfield-dev-1293-e1246c8824a83c760b962d970625d356b711ae8e-x86.apk)
 - [**arm64_v8a**](https://download.opengis.ch/qfield/ci-builds/qfield-dev-1293-e1246c8824a83c760b962d970625d356b711ae8e-arm64_v8a.apk)
 - [**x86_64**](https://download.opengis.ch/qfield/ci-builds/qfield-dev-1293-e1246c8824a83c760b962d970625d356b711ae8e-x86_64.apk)

**Comment by qfield-fairy** [comment]
Uploaded test apks for
 - [**armv7**](https://download.opengis.ch/qfield/ci-builds/qfield-dev-1293-106a5322390ffe86868ef085cbb0aa3e20ab31d0-armv7.apk)
 - [**x86**](https://download.opengis.ch/qfield/ci-builds/qfield-dev-1293-106a5322390ffe86868ef085cbb0aa3e20ab31d0-x86.apk)
 - [**arm64_v8a**](https://download.opengis.ch/qfield/ci-builds/qfield-dev-1293-106a5322390ffe86868ef085cbb0aa3e20ab31d0-arm64_v8a.apk)
 - [**x86_64**](https://download.opengis.ch/qfield/ci-builds/qfield-dev-1293-106a5322390ffe86868ef085cbb0aa3e20ab31d0-x86_64.apk)

**Review by suricactus — APPROVED** [review]

File: scripts/ci/update-translations.sh [hunk 2]
```diff
@@ -3,7 +3,7 @@
 DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null && pwd )"
 source ${DIR}/../version_number.sh
 
-lupdate -recursive ${DIR} -ts ${DIR}/i18n/qfield_en.ts
+lupdate -recursive ${CI_BRANCH} -ts ${CI_BRANCH}/i18n/qfield_en.ts
```
Comment (m-kuhn) [inline scripts/ci/update-translations.sh:6]: What's this change here for?

Comment (signedav) [reply scripts/ci/update-translations.sh:6]: Otherwise it's looking for the files in `scripts/ci/`, no?
Could be `${DIR}/../..` as well...

**Review by m-kuhn — APPROVED** [review]

File: scripts/ci/update-translations.sh (hunk 2 above)
Comment (signedav) [reply scripts/ci/update-translations.sh]: `CI_BRANCH` is wrong of course...

**Comment by VxTedxV** [comment]
Some lines stay untranslated in v1.7.1 like "map themes", "you are now in digitize mode", "you are now in browse mode", "show on map canvas", expand group", "expand layer item", "zoom to layer", on welcome screen too...
I thought it was exhaustive in #987

**Comment by signedav** [comment]
Hm... Strings like "changelog" now show up in Transifex (but are still untranslated) so this part is working.

But the mentioned ones like "show on map canvas " are translated in Transifex but still in english on the app.
 Any idea @m-kuhn @suricactus ?

**Comment by m-kuhn** [comment]
I just tested it in German with a local build with translations fetched right now and "show on map canvas" shows up translated / as expected.
Either this is an android / CI build issue or the test results metioned above are from older builds.

**Comment by signedav** [comment]
The test results are from 1.7.1 and from a master build made by:
```
ARCH=arm64-v8a
QFIELD_SDK_VERSION=20200914
sudo docker run -it -v $(pwd):/usr/src/qfield -e "BUILD_FOLDER=build-${ARCH}" -e ARCH -e STOREPASS -e KEYNAME -e KEYPASS -e PKG_NAME -e APP_NAME -e APP_ICON -e "APP_VERSION=1.6.0" -e "APP_VERSION_CODE=161" -e APP_VERSION_STR opengisch/qfield-sdk:${QFIELD_SDK_VERSION} /usr/src/qfield/scripts/docker-build.sh
```
Did I missed something? I'm not very familiar with it and this PR scoped only on the issue, that the ts-files have not been updated (and with it Transifex) what works fine now.

**Comment by m-kuhn** [comment]
I don't think the command will do a `tx pull` you have to do that by hand.

**Comment by VxTedxV** [comment]
> I just tested it in German with a local build with translations fetched right now and "show on map canvas" shows up translated / as expected.
> Either this is an android / CI build issue or the test results metioned above are from older builds.

Do you need more info from me ? Don't know if that could help : Android 9, armv7, french language expected in translations

**Pull request merged**